db/models.py

```diff
@@ -1 +1,43 @@
 from django.db import models
+
+
+class Race(models.Model):
+    races = [
+        ("e", "elf"),
+        ("d", "dwarf"),
+        ("h", "human"),
+        ("o", "ork")
+    ]
+    name = models.CharField(unique=True, max_length=255, choices=races)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(unique=True, max_length=255)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(
+        Race,
+        on_delete=models.CASCADE
+    )
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255)
+    description = models.TextField(blank=True, null=True)
+
+
+class Player(models.Model):
+    nickname = models.CharField(unique=True, max_length=255)
+    email = models.EmailField(max_length=255)
+    bio = models.TextField()
+    race = models.ForeignKey(
+        Race,
+        on_delete=models.CASCADE
+    )
+    guild = models.ForeignKey(
+        Guild,
+        on_delete=models.CASCADE,
+        blank=True,
+        null=True
+    )
```

Comment:
this field cannot be null or blank, because the player will be deleted when Guild is deleted. You should set another `on_delete` value, so that player will be not deleted. `blank=True` is unnecessary in any cases